komodor/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Track changes across your entire K8s landscape and stack",
+    "title": "Komodor Automation",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Main services screen.",
+        "image_url": "images/komodor_screen_01.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Service view events timeline and related services.",
+        "image_url": "images/komodor_screen_02.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Services view reviewing a deployment and its changes.",
+        "image_url": "images/komodor_screen_03.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Reviewing a deployment's replicaset and its pods and logs",
+        "image_url": "images/komodor_screen_04.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Events timelines for multiple clusters and deployments.",
+        "image_url": "images/komodor_screen_05.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Events time lines for multiple clusters and deployments.",
+        "image_url": "images/Komodor_screen_06.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Events time lines for multiple clusters and deployments.",
```

Comment:
This caption is the same as the one above - we recommend unique captions for each image in the media carousel.